Src/FluentAssertions/Polyfill/StringBuilderExtensions.cs

```diff
@@ -1,5 +1,8 @@
-﻿using System.Collections.Generic;
+﻿#if NET47 || NETSTANDARD2_0 || NETSTANDARD2_1
```

Comment:
FYI: these files are excluded from net6 and netstandard21 in 
https://github.com/fluentassertions/fluentassertions/blob/9d0daeea0cb24ef1c48c9c8f6386d5fd53549893/Src/FluentAssertions/FluentAssertions.csproj#L58C17-L59

https://github.com/fluentassertions/fluentassertions/blob/9d0daeea0cb24ef1c48c9c8f6386d5fd53549893/Src/FluentAssertions/FluentAssertions.csproj#L69C10-L69C10

Reply:
Ah, I was looking, how this worked before without excluding .NET6 😃

Reply:
Should this then be:

```suggestion
#if NET47 || NETSTANDARD2_0
```

Reply:
No, the `StringBuilderExtensions` were only excluded from .NET 6.